### PR TITLE
Add support for volume ownership and permissions modification

### PR DESCRIPTION
**Pull request by sergeyberezansky**

### CODE REVIEW

File: examples/dynamic_api/storageclass-wekafs-dir-api.yaml [hunk 1]
```diff
@@ -9,8 +9,15 @@ allowVolumeExpansion: true
 parameters:
   volumeType: dir/v1
   filesystemName: default
-  # if the values below are not set, quota will not be enforced for this storage class
+  # optional parameters setting UID, GID and permissions on volume
+  # UID of the volume owner, default 0 (root)
+  ownerUid: "1000"
```
Comment (tigrawap) [inline examples/dynamic_api/storageclass-wekafs-dir-api.yaml:14]: maybe put it as a separate example or comment out?
As of right now, this basically makes the default example not usable

Comment (sergeyberezansky) [reply examples/dynamic_api/storageclass-wekafs-dir-api.yaml]: Done

File: examples/dynamic_api/storageclass-wekafs-dir-api.yaml [hunk 2]
```diff
@@ -9,8 +9,18 @@ allowVolumeExpansion: true
...
   volumeType: dir/v1
   filesystemName: default
-  # if the values below are not set, quota will not be enforced for this storage class
 
+  # optional parameters setting UID, GID and permissions on volume
+  # UID of the volume owner, default 0 (root)
+  #ownerUid: "1000"
+
+  # GID of the volume owner, default 0 (root)
+  ownerGid: "1000"
```
Comment (tigrawap) [inline examples/dynamic_api/storageclass-wekafs-dir-api.yaml:18]: now it is inconsistent... either all permissions related should present or just one
I prefer all of them commented out, which is standard approach

Comment (sergeyberezansky) [reply examples/dynamic_api/storageclass-wekafs-dir-api.yaml]: have no idea how I missed it.

**Pull request merged**